tests/functional/test_reinstall.py

```diff
@@ -149,7 +150,7 @@ def test_cylc_install_files(fixture_install_flow, file_, expect):
         ),
         (
             'opt/rose-suite-cylc-install.conf', (
-                '# This file records CLI Options.\n\n'
+                '# This file records CLI Options.{version_info}\n'
```

Comment:
This isn't a format string.

The test still passes, so the `version_info` is presumably not require here?

Reply:
No, it's not a f-string, but I use format on it later to insert `version_info`:

```python
expect = expect.format(version_info=version_info)
```